config-templates/module_elixir.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * The config template for module ELIXIR
+ */
+$config = array(
+
+    /**
+     * The clientId from CSC_MFA server
+     */
+    'clientId' => '',
+
+    /**
+     * The clientSecret from CSC_MFA server
+     */
+    'clientSecret' => '',
+
+    /**
+     * List of requested scopes
+     */
+    'reguestedScopes' => [],
+
+    /**
+     * The authorizationEndpoint of CSC_MFA server
+     */
+    'authorizationEndpoint' => '',
```

Comment:
Shouldn't we use just one URL? All endpoints can anyway be parsed from the _issuer_/.well-known/openid-configuration.